### PR TITLE
Add support to read h5 files

### PR DESCRIPTION
**Pull request by ntadimeti**

### CODE REVIEW

File: scripts/main.py [hunk 1]
```diff
@@ -358,58 +358,70 @@ def main():
...
-
-        args.train_files = train_files
-        args.val_files = val_files
+        
+
+        # If h5 files are provided, load them.
+        if args.train_h5_files is not None:
+            args.train_files = gather_files_from_cmdline(
+                    args.train_h5_files,
+                    extension=".bw")
```
Comment (avantikalal) [inline scripts/main.py:367]: extension = ".h5"?

File: scripts/main.py [hunk 2]
```diff
@@ -358,58 +358,70 @@ def main():
...
+            args.val_files = gather_files_from_cmdline(
+                    args.val_h5_files,
+                    extension=".bw")
```
Comment (avantikalal) [inline scripts/main.py:370]: extension = ".h5"?

File: configs/infer_config.yaml [hunk 3]
```diff
@@ -48,3 +48,7 @@ batches_per_worker: 16
 result_fname: infer
 gen_bigwig: False
 deletebg: False
+
+# ADVANCED OPTIONS
```
[the remark this answers is not in the document]
Comment (ntadimeti) [reply configs/infer_config.yaml:50]: done.

File: scripts/main.py (hunk 2 above)
Comment (ntadimeti) [reply scripts/main.py]: done.

File: scripts/main.py (hunk 1 above)
Comment (ntadimeti) [reply scripts/main.py]: done.

**Pull request merged**